attributevalue/src/main/scala/com/hiya/alternator/syntax/package.scala

```diff
@@ -19,5 +19,9 @@ package object syntax {
     def between[T : ScalarDynamoFormat](lower: T, upper: T): RKCondition.BETWEEN[T] = RKCondition.BETWEEN(lower, upper)
   }
 
-
+  object cond {
+    def attr[T](name: String): ConditionExpression.Path[T] = ConditionExpression.Attr(name)
```

Comment:
I'd put it to top level. Or maybe event create a stringcontext for it.

Reply:
That means we have a name collision is we ever want to support partial updates via update expressions, although in that case probably we'd share some implementation between those and condition expressions. I'll put it to the top level. I'd avoid StringContexts for this, it only saves a pair of parens and with `attr` being a function you can use explicit type ascriptions like `attr[String]("Price")`.